app/src/main/java/com/android/sample/ui/listActivities/ViewListActivities.kt

```diff
@@ -92,14 +90,11 @@ fun ListActivitiesScreen(
 ) {
   val context = LocalContext.current
   val uiState by viewModel.uiState.collectAsState()
-  var selectedIndex by remember { mutableIntStateOf(0) }
-  val all = "ALL"
-  val typesToString = types.map { it.name }
-  val options = listOf(all) + typesToString
+  val options = categories.map { it.name }
   val profile = profileViewModel.userState.collectAsState().value
   var searchText by remember { mutableStateOf("") }
   var showFilterDialog by remember { mutableStateOf(false) }
-
+  val checkedList = remember { mutableStateListOf<Int>() }
```

Comment:
Okay so this is the list that keeps the filtering choices. I guess we could have placed this in a view model to keep it even when switching/recomposing the screen.